src/vs/workbench/browser/parts/editor/tabsTitleControl.ts

```diff
@@ -663,6 +713,13 @@ export class TabsTitleControl extends TitleControl {
 			this.updateTabsScrollbarSizing();
 		}
 
+		// Update tabs sizing
+		if (oldOptions.tabSizingFixedMaxWidth !== newOptions.tabSizingFixedMaxWidth ||
+			oldOptions.tabSizing !== newOptions.tabSizing
+		) {
+			this.updateTabSizing();
+		}
```

Comment:
```suggestion
		// Update tabs sizing
		if (
			oldOptions.tabSizingFixedMaxWidth !== newOptions.tabSizingFixedMaxWidth ||
			oldOptions.tabSizing !== newOptions.tabSizing
		) {
			this.updateTabSizing();
		}
```

Reply:
(sorry in my last suggestion I had forgotten that I actually wanted this formatting)